[EDU-4928] feature: add Docusaurus Blog template guide (#1119)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/docusaurus-blog-template.mdx

---

> title: How to deploy the Docusaurus Blog template

Author: ```suggestion
title: How to deploy the Docusaurus Blog Starter Kit
```

description: Create and manage a blog page with support to Markdown, MDX, and React components, based on Docusaurus.
meta_tags: templates, guides, Azion Marketplace, Docusaurus, React
namespace: docs_guides_docusaurus_docusaurus_blog_template

> permalink: /documentation/products/guides/docusaurus-blog-template/

Author: ```suggestion
permalink: /documentation/products/guides/docusaurus-blog-starter-kit/
```

---

import Tag from 'primevue/tag';

<Tag severity="info" client:only="vue">
Preview
</Tag>

> The **Docusaurus Blog** template streamlines the creation and management of a blog page with support to Markdown and MDX, making it easy to write content with rich media and React components.

Author: ```suggestion
The **Docusaurus Blog Starter Kit** template streamlines the creation and management of a blog page with support to Markdown and MDX, making it easy to write content with rich media and React components.
```

> The **Docusaurus Blog Starter Kit** template streamlines the creation and management of a blog page with support to Markdown and MDX, making it easy to write content with rich media and React components.

Author: ```suggestion
The **Docusaurus Blog Starter Kit** streamlines the creation and management of a blog page with support to Markdown and MDX, making it easy to write content with rich media and React components.
```

> Docusaurus is a static-site generator, building a single-page application with fast client-side navigation and leveraging React to make the site interactive. It's ideal for documentation features and other cases, including website, product pages, blog, and landing pages.

Reviewer: ```suggestion
Docusaurus is a static-site generator, that builds a single-page application with fast client-side navigation and leverages React to make the site interactive. It's ideal for documentation features and other cases, including website, product pages, blog, and landing pages.
```

:::

---

## Requirements

- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
  - Every push will be deployed automatically to the main branch in this repository to keep your project updated.
- Enable [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/) in your account. 
    - To do so, go to the [Billing & Subscriptions](/en/documentation/products/guides/billing-and-subscriptions/#subscriptions) section and activate the switch for each module.
    - If this module isn't activated, the execution will fail and a log explaining the reason will be printed.
    - If this module is activated, deploying this template could generate usage-related costs. Check the [pricing page](/en/documentation/products/pricing/) for more information.

---

## Getting the template

> To start using the **Docusaurus Blog** template:

Author: ```suggestion
To start using the **Docusaurus Blog Starter Kit**:
```

1. Access [Console](https://console.azion.com).
2. Click the **+ Create** button on the homepage.
3. In the modal, select the **Templates** option. 

> 4. Browse and select the **Docusaurus Blog** card.

Author: ```suggestion
4. Browse and select the **Docusaurus Blog Starter Kit** card.
```
